static/gsApp/components/ai/AiSetupDataConsent.tsx

```diff
@@ -43,75 +71,175 @@ function AiSetupDataConsent({groupId}: AiSetupDataConsentProps) {
           )}
         </Paragraph>
-      </ConsentItem>
-      <ConsentItem>
-        <ConsentTitle>{t('Root Cause Analysis')}</ConsentTitle>
-        <Paragraph>
-          {t('A streamlined, collaborative workflow to find the root cause.')}
-        </Paragraph>
-      </ConsentItem>
-      <ConsentItem>
-        <ConsentTitle>{t('Solutions & Code Changes')}</ConsentTitle>
-        <Paragraph>
-          {t('Proposed fixes with test cases, ready to merge as draft pull requests.')}
-        </Paragraph>
-      </ConsentItem>
-      <ButtonWrapper>
-        <Button
-          priority="primary"
-          onClick={() => autofixAcknowledgeMutation.mutate()}
-          disabled={autofixAcknowledgeMutation.isPending}
-          analyticsEventKey="gen_ai_consent.in_drawer_clicked"
-          analyticsEventName="Gen AI Consent: Clicked In Drawer"
-          analyticsParams={{
-            is_first_user_in_org:
-              !autofixSetupData?.setupAcknowledgement.orgHasAcknowledged,
-          }}
-          size="sm"
-        >
-          {autofixAcknowledgeMutation.isPending ? (
-            <StyledLoadingIndicator size={14} />
-          ) : autofixSetupData?.setupAcknowledgement.orgHasAcknowledged ? (
-            t('Try Seer')
+        <BulletList>
+          <li>{t('Issue Triage')}</li>
+          <li>{t('Root Cause Analysis')}</li>
+          <li>{t('Solutions & Code Changes')}</li>
+        </BulletList>
+        {shouldShowBilling ? (
+          isTouchCustomer ? (
+            <TouchCustomerMessage>
+              {t('Contact your customer success manager to get access to Seer.')}
+            </TouchCustomerMessage>
           ) : (
-            t('Enable Seer')
-          )}
-        </Button>
-        {autofixAcknowledgeMutation.isError && (
-          <ErrorText>{t('Something went wrong.')}</ErrorText>
+            <Fragment>
+              <ButtonWrapper>
+                {canStartTrial ? (
+                  <StartTrialButton
+                    organization={organization}
+                    source="ai-setup-consent"
+                    requestData={{
+                      productTrial: {
+                        category: DataCategory.SEER_AUTOFIX,
+                        reasonCode: 1001,
+                      },
+                    }}
+                    busy={autofixAcknowledgeMutation.isPending}
+                    handleClick={() => autofixAcknowledgeMutation.mutate()}
+                    size="md"
+                    priority="primary"
+                  >
+                    {t('Try Seer for Free')}
+                  </StartTrialButton>
+                ) : hasSeerButNeedsPayg ? (
+                  <Flex gap={space(2)} column>
+                    <ErrorText>
+                      {t(
+                        "You've run out of pay-as-you-go budget. Please add more to keep using Seer."
+                      )}
```

Comment:
```suggestion
                      {tct(
                        "You've run out of [budgetTerm] budget. Please add more to keep using Seer.",
                        {budgetTerm: subscription.planDetails.budgetTerm}
                      )}
```
we have different names based on the plan tier 😅